File: numbers/numbers.ts
function main() {
  const num: number = -8.7;
  const base = 2;
  const exp = 3;

  // Absolute value
  const abs = Math.abs(num);
  console.log("Abs:", abs);

  // Rounding
  const round = Math.round(num);
  console.log("Round:", round);

  // Truncate
  const trunc = Math.trunc(num);
  console.log("Trunc:", trunc);

  // Ceiling
  const ceil = Math.ceil(num);
  console.log("Ceil:", ceil);

  // Floor
  const floor = Math.floor(num);
  console.log("Floor:", floor);

  // Square root
  const sqrt = Math.sqrt(64);
  console.log("Sqrt(64):", sqrt);

  // Power
  const power = Math.pow(base, exp);
  console.log("Pow(2, 3):", power);

  // Random number
  const random = Math.random();
  console.log("Random (0~1):", random);
}

main();

File: numbers/numbers.go
package main

import (
	"fmt"
	"math"
	"math/rand"
)

func main() {
	num := -8.7
	base := 2.0
	exp := 3.0

	// Absolute value
	abs := math.Abs(num)
	fmt.Println("Abs:", abs)

	// Rounding
	round := math.Round(num)
	fmt.Println("Round:", round)

	// Truncate
	trunc := math.Trunc(num)
	fmt.Println("Trunc:", trunc)

	// Ceiling
	ceil := math.Ceil(num)
	fmt.Println("Ceil:", ceil)

	// Floor
	floor := math.Floor(num)
	fmt.Println("Floor:", floor)

	// Square root
	sqrt := math.Sqrt(64)
	fmt.Println("Sqrt(64):", sqrt)

	// Power
	power := math.Pow(base, exp)
	fmt.Println("Pow(2, 3):", power)

	// Random number
	random := rand.Float64()
	fmt.Println("Random (0~1):", random)
}
